new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -63,7 +114,7 @@ public void testCreatingDomain() {
         .kind("Domain")
         .metadata(metadata)
         .spec(domainSpec);
-    boolean success = assertDoesNotThrow(
+    success = assertDoesNotThrow(
```

Comment:
Add message string for when it does throw an exception. What should the engineer debugging this look for as the problem?

Reply:
Added message